cli/test-utils/src/script.rs

```diff
@@ -88,17 +88,17 @@ impl ScriptTester {
         }
     }
 
-    fn link_testdata(current_dir: &Path) -> eyre::Result<()> {
+    fn copy_testdata(current_dir: &Path) -> eyre::Result<()> {
         let testdata = format!("{}/../../testdata", env!("CARGO_MANIFEST_DIR"));
-        std::fs::hard_link(
+        std::fs::copy(
```

Comment:
Hard links from some non-temp directory to a temp directory is considered a violation of filesystem boundaries on Windows, so we have to copy instead 🙄